phantomgames/Search/SearchGameViewModel.swift

```diff
@@ -17,12 +17,12 @@ class SearchGameViewModel {
     // MARK: Variables
     
```

Comment:
On line 50 add a line break after the mark for consistency otherwise remove line breaks after MARKS